packages/js/src/client.ts

```diff
@@ -3,6 +3,12 @@ import { users } from './users';
 import { Batch, createBatchKey } from './batch';
 import HTTPClient, { ClientOptions } from './httpClient';
 
+/**
+ * Axiom's BaseClient that works as a wrapper around the HTTPClient
+ * 
+ * @param options - The options passed to the client, check: {@link ClientOptions}
+ * 
+ */
```

Comment:
Does this need docs as it's not exported?

Reply:
removed